gobblin-data-management/src/main/java/gobblin/data/management/copy/publisher/CopyDataPublisher.java

```diff
@@ -218,9 +224,11 @@ private void publishFileSet(CopyEntity.DatasetAndPartition datasetAndPartition,
       datasetUpstreamTimestamp = 0;
     }
 
+    additionalMetadata.put(SlaEventKeys.SOURCE_URI, this.state.getProp(SlaEventKeys.SOURCE_URI));
+    additionalMetadata.put(SlaEventKeys.DESTINATION_URI, this.state.getProp(SlaEventKeys.DESTINATION_URI));
     CopyEventSubmitterHelper.submitSuccessfulDatasetPublish(this.eventSubmitter, datasetAndPartition,
-        Long.toString(datasetOriginTimestamp), Long.toString(datasetUpstreamTimestamp));
-  }
+        Long.toString(datasetOriginTimestamp), Long.toString(datasetUpstreamTimestamp), additionalMetadata);
```

Comment:
As per the WIMD team request, I added Hive metastore information in PartitionDropSucceed and TableDropSucceed events only.
Isn't hive metastore information deduced from source and target URIs? As per my understanding, it is a one-to-one mapping.